src/job/job.spec.js

```diff
@@ -146,10 +171,28 @@ describe("job", () => {
           },
         },
       });
-      expect(await xo.call("job.get", { id })).toMatchSnapshot({
+
+      const job = await xo.call("job.get", { id });
+      expect(job).toMatchSnapshot({
         id: expect.any(String),
+        paramsVector: expect.any(Object),
         userId: expect.any(String),
       });
+      expect(job.paramsVector).toEqual({
+        items: [
+          {
+            type: "set",
+            values: [
+              {
```

Comment:
It can be mutualized.

Reply:
I thought that as it's different than the others, it was simpler to let it like that, and mutualise at best for others. What do you think ? Should I change it anyway ?